assets/panel/debugger.properties

```diff
@@ -470,7 +470,7 @@ sourceSearch.search=Search Sources…
 
 # LOCALIZATION NOTE (sourceSearch.noResults): The center pane Source Search
 # message when the query did not match any of the sources.
-sourceSearch.noResults=No files matching %S found
```

Comment:
hmm, its not being used anywhere else -- why does this need to changed to have a new key?